app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -40,7 +41,18 @@
       android:text="@={viewModel.answerText}"
       android:textColor="@color/oppiaPrimaryText"
       android:textColorHint="@color/edit_text_hint"
-      android:textSize="16sp"
-      android:textStyle="italic" />
-  </FrameLayout>
+      android:textSize="14sp"
+      android:textStyle="italic"
+      app:textChangedListener="@{viewModel.passwordTextWatcher}" />
+
+    <TextView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="8dp"
+      android:layout_marginEnd="8dp"
+      android:fontFamily="sans-serif"
+      android:text="@{viewModel.pendingAnswerError}"
```

Comment:
Error message databinding

Reply:
I think you need to call this using functional syntax since it's a function.

What's the error that you're seeing?

Reply:
Done